packages/platform-browser/src/hydration.ts

```diff
@@ -70,6 +71,15 @@ export function withHttpTransferCacheOptions(
       HydrationFeatureKind.HttpTransferCacheOptions, ɵwithHttpTransferCache(options));
 }
 
+/**
+ * Enables experimental support for hydrating i18n blocks.
```

Comment:
I'd propose dropping "experimental", since it may be confused with `@experimental` APIs.
```suggestion
 * Enables support for hydrating i18n blocks.
```